test/ocsp/helper/helper.go

```diff
@@ -50,15 +51,14 @@ var DefaultConfig = Config{
 	expectReason:       *expectReason,
 }
 
-var configFromFlags Config
-var isConfigPopulated bool = false
+var configFromFlagsMemo sync.Once
 
 // ConfigFromFlags returns a Config whose values are populated from
 // any command line flags passed by the user, or default values if not passed.
 func ConfigFromFlags() Config {
-	if !isConfigPopulated {
+	return configFromFlagsMemo.Do(func() {
```

Comment:
Oh whoops, I think you just want to stick the flag.Parse() in here, the return won't actually return from `ConfigFromFlags`, and you're using Do() as a value, which doesn't actually return anything.

Reply:
Yeah I had it like that and then thought "wait, I can just cache the whole config object instead of just the flags! genius!" and then forgot to rebuild :P

Reply:
I think the `flag.Parse()` and the assignment to a global `configFromFlags` both need to go inside the Do func.

Reply:
I removed the assignment to a global configFromFlags. It's no more expensive to rebuild a Config from the individual flag values on every call that it is to copy-assign a new Config from the saved configFromFlags. And none of the flags are exported, so no one should be messing with them at runtime either.